CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ FEATURES:
  * **Consul Namespaces (Enterprise)**: Added support for Consul Namespaces [[GH-10235](https://github.com/hashicorp/nomad/pull/10235)]
  * **CSI Volume Create and Snapshot**: Added support for creating, deleting, listing, and snapshotting volumes managed by Container Storage Interface (CSI) plugins. [[GH-8212](https://github.com/hashicorp/nomad/issues/8212)]
  * **Licensing (Enterprise)**: Support loading Enterprise license from disk or environment. [[GH-10216](https://github.com/hashicorp/nomad/issues/10216)]
+ * **`nomad ui -authenticate`**: Added the authenticate flag to open the UI with the ACL token. [[GH-10097](https://github.com/hashicorp/nomad/issues/10097)] [[GH-10066](https://github.com/hashicorp/nomad/pull/10066)]
```

Comment:
We had this under `IMPROVEMENTS` but yeah this is a nice feature... let's put it here. Can you remove this line below?
> * cli: Added `nomad ui -authenticate` flag to generate a one-time token for authenticating to the web UI when ACLs are enabled. [[GH-10097](https://github.com/hashicorp/nomad/issues/10097)]

Reply:
Done, thank you!